packages/design-system/src/components/N8nInfoAccordion/InfoAccordion.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+	<div :class="['accordion', $style.container]" >
+		<div :class="{[$style.header]: true, [$style.expanded]: expanded}" @click="toggle">
+			<n8n-text color="text-base" size="small" align="left" bold>{{ title }}</n8n-text>
+
+			<font-awesome-icon
+				:class="$style.chevron"
+				:icon="expanded? 'chevron-up' : 'chevron-down'"
+				bold
+			/>
+
+		</div>
+		<div v-if="expanded" :class="{[$style.description]: true, [$style.collapsed]: !expanded}" @click="onClick">
+			<n8n-text color="text-base" size="small" align="left">
+				<span v-html="description"></span>
+			</n8n-text>
+		</div>
+	</div>
+</template>
+
+<script>
+export default {
+	name: 'n8n-info-accordion',
+	props: {
+		title: {
+			type: String,
+		},
+		description: {
+			type: String,
+		},
+	},
+	mounted() {
+		this.$on('expand', () => {
```

Comment:
Can this listener be handled the usual way (`v-on` that triggers method)?

Reply:
no because this is waiting for events from parent